DOC-5589/updated-config-database-1.2-1.5 (#1593)
corda/corda-docs-portal: content/en/platform/corda/1.2/cenm/config-database.md

# CENM Database Configuration

A persistent layer must be available for the Identity Manager and Network Map Components to function. This is described in
their respective configs with the following configuration states.

* **database**:
Database configuration. 

* **runMigration**:
Create or upgrade the database schema (database objects like tables, indices) to the current version of the service. If it is set to false, and the database schema is up to date, the service will validate.

* **initialiseSchema**:
*(Deprecated)* Automatically creates the tables required by the CENM component.

> {{< note>}} This property was used for H2 database only, its ignored since CENM 1.2 and its replaced by `runMigration`.{{< /note>}}

Reviewer: ```suggestion
{{< note>}} This property was used for H2 database only, it has been ignored since CENM 1.2 and it has been replaced by `runMigration`.{{< /note>}}
```

* **jdbcDriver**:

> Path to the jar file containing the specific JDBC driver.

Reviewer: ```suggestion
Path to the JAR file containing the specific JDBC driver.
```

* **driverClassName**:
See the specific JDBC driver documentation [here](https://www.oracle.com/java/technologies/javase/javase-tech-database.html).

* **url**:

> Location of the Database on the network.

Reviewer: ```suggestion
Location of the database on the network.
```
